invenio_app_ils/static_pages/search_guide.html

```diff
@@ -0,0 +1,636 @@
+
+
+<h3 id="phrasesearch">Phrase search</h3>
+
+<p><strong>Example:</strong> <a href="/search?q=%22dark%20matter%22"><code>"dark matter"</code></a></p>
+
+<p>Results will match records with the <em>phrase</em> <code>dark matter</code> in <em>any field</em>.</p>
+
+<h3 id="fieldsearch">Field search</h3>
+
+<p><strong>Example:</strong> <a href="/search?q=title:matter"><code>title:matter</code></a></p>
+
+<p>Results will match records with the <em>term</em> <code>matter</code> in the <em>field</em> <code>title</code>. If you want to search for multiple terms in the title you must <strong>group the terms</strong> using parenthesis:</p>
+
+<p><strong>Example:</strong> <a href="/search?q=title:(dark%20matter)"><code>title:(dark matter)</code></a></p>
+
+<p>See the field reference below for the full list of fields you can search.</p>
+
+<h3 id="combinedsimplephraseorfieldsearch">Combined simple, phrase or field search</h3>
+
+<p><strong>Example:</strong> <a href="/search?q=%2Btitle:%22dark%20matter%22%20-title:experiment"><code>+title:"dark matter" -title:experiment</code></a> or <a href="/search?q=title:(-dark%20%2Bmatter)"><code>title:(-dark +matter)</code></a></p>
+
+<p>You can combine simple, phrase and field search to construct advanced search queries.</p>
+
+<h3 id="rangesearch">Range search</h3>
+
+<p><strong>Example:</strong> <a href="/search?q=publication_year:[2020 TO 2021]"><code>publication_year:[2020 TO 2021]</code></a> (note, you must capitalize <code>TO</code>).</p>
+
+<p>Results will match any record with a publication date between 2020-01-01 and 2021-12-31 (both dates inclusive).</p>
+
+<p>Examples of other ranges:</p>
+
+<ul>
+<li><code>publication_year:{* TO 2020}</code> - All days until 2020.</li>
+
+<li><code>publication_year:[2020 TO *]</code> - All days from 2020.</li>
+
+</ul>
+
+<h3 id="phrasesearch">Examples of searches on literature</h3>
+
+<h4>Identifiers</h4>
+
+<strong>Example:</strong>
+
+<ul>
+<li><a href="/search?q=9814749133"><code>"9814749133"</code></a> or <a href="/search?q=9789814749138"><code>"9789814749138"</code></a> - ISBN 10 or 13</li>
+<li><a href="/search?q=10.1016\/j.physletb.208020"><code>"10.1016/j.physletb.208020"</code></a> - DOI</li>
+<li><a href="/search?q=1745-2481"><code>"1745-2481"</code></a> - ISSN</li>
+  <li><a href="/search?q=NF-EN-81-80"><code>"NF-EN-81-80”</code></a> - Standard number</li>
+</ul>
+
+<p>Results will match exactly the provided identifier.</p>
+
+<h4>Author and affiliation</h4>
+
+<strong>Example:</strong>
+
+<ul>
+<li><a href="/search?q=authors.full_name:(Albert%20Einstein)"><code>authors.full_name:(Albert Einstein)</code></a> - literature from a specific author</li>
+<li><a href="/search?q=authors.affiliations.name:CERN"><code>authors.affiliations.name:CERN</code></a> - literature whose author is affiliated to CERN</li>
```

Comment:
should we remove CERN references? ping @ntarocco

Reply:
I guess we can keep them, it doesn't hurt? I mean, ideally if we have some demo data with some affiliations, probably better to put that so that the search example works nicely.